src/api-server.ts

```diff
@@ -271,17 +273,23 @@ async function createTopicWithRskAddress (call: any) {
     }
     console.log(`createTopicWithRskAddress ${JSON.stringify(call.request)} `)
     try {
-        console.log(`locatePeerID ${JSON.stringify(call.request.address)} `)
-        const address = await dht.getPeerIdByRskAddress(call.request.address);
-        console.log("address", address)
-        if (address === null) {
+        const rskAddress = call.request.address;
+        console.log(`locatePeerID ${JSON.stringify(rskAddress)} `)
+        const peerId = await dht.getPeerIdByRskAddress(rskAddress);
+        console.log("address", peerId)
+        if (peerId === null) {
```

Comment:
I know the structure of this part of the code was already done this way. But I just realized this check is unnecessary, as an exception is already thrown if the rskAddress is not found in the DHT.

Reply:
good catch (pun intended)